--- ifup-tunnel 2019-12-18 18:47:31.582684959 +0300
+++ ifup-tunnel 2019-12-17 01:43:10.336545205 +0300
@@ -71,6 +71,10 @@
     ${PEER_OUTER_IPADDR:+remote "$PEER_OUTER_IPADDR"} \
     ${KEY:+key "$KEY"} ${TTL:+ttl "$TTL"}
 
+if [[ "${MULTICAST:-NOP}" =~ [Yy][Ee][Ss] ]]; then
+    /sbin/ip link set "$DEVICE" multicast on
+fi
+
 if [ -n "$MTU" ]; then
     /sbin/ip link set "$DEVICE" mtu "$MTU"
 fi
